content/en/docs/setup/install/operator/index.md

```diff
@@ -106,20 +112,26 @@ Features can be enabled or disabled, which enables or disables all of the compon
 Namespaces can also be set at the feature level, which installs all of the feature's components into the feature
 namespace.
 
-## Modify a feature or component setting
+### Configure the feature or component settings
+
+After you indentify the name of the feature or component from the previous table, you can use the API to set the values using the `--set` flag, or create an overlay file and pass it with the `--filename` flag. In general, the `--set` flag works well for customizing a few parameters, while overlay files are better for more extensive customization, or tracking configuration changes.
+
+For example, to enable Pilot, which is part of the Traffic Management feature, use this command:
+{{< text bash >}}
+$ istioctl experimental manifest apply --set trafficManagment.components.pilot.enabled=true
+{{< /text >}}
```

Comment:
I'm not sure this example adds much here, I think it's better to stick with just turning off telemetry below, using either --set or --filename.

Reply:
I think you'll want this because it shows using the info from the table above to configure a component, and then doing an action at the feature level is also shown after it.

Reply:
In that case, let's create a more realistic example (Pilot is almost always on) and put it below the telemetry example e.g. 
istioctl experimental manifest apply --set security.components.nodeAgent.enabled=true

Reply:
done